platforms/stm32_common/usb/cdc.c

```diff
@@ -9,7 +9,7 @@
 #include "platform/usb/cdc.h"
 #include "platform/usb/usb.h"
 #include "usb/core.h"
-#include "stm32l4xx_hal.h"
+#include "stm32_hal.h"
 #include "cdc_internal.h"
```

Comment:
This is not sorted alphabetically anymore :<

Reply:
This is a stupid rule anyway. Sorting it alphabetically breaks the include depencency.

Reply:
```
In file included from platforms/stm32_common/usb/cdc.c:6:0:
platforms/stm32_common/usb/cdc_internal.h:6:2: error: unknown type name 'USBD_HandleTypeDef'
  USBD_HandleTypeDef *p_usbd;
  ^~~~~~~~~~~~~~~~~~
platforms/stm32_common/usb/cdc.c: In function 'usb_cdc_init':
platforms/stm32_common/usb/cdc.c:261:14: warning: assignment from incompatible pointer type [-Wincompatible-pointer-types]
  SELF.p_usbd = p_data->p_usbd;
```

Reply:
then cdc internal should include the hal, can't have order-dependant includes if we can help it, and if we don't have some kind of rule, be it A-Z sort or length then it's just total chaos